include/lsst/afw/fits.h

```diff
@@ -510,6 +511,19 @@ class Fits {
 
 #endif // !SWIG
 
+//@{
+/// @brief Read FITS header
+///
+/// If 'strip' is true, common FITS keys that usually have non-metadata intepretations
+/// (e.g. NAXIS, BITPIX) will be ignored.
+///
+/// Includes support for the INHERIT convention: if 'INHERIT = T' is in the header, the
+/// PHU will be read as well, and nominated HDU will override any duplicated values.
+PTR(daf::base::PropertyList) readMetadata(std::string const & fileName, int hdu=0, bool strip=false);
```

Comment:
This seems like a very logical set of new functions.

I'm a bit disappointed that we can't just return a PropertyList instead of a pointer to a property list. I assume the pointer is used to avoid copying the data (though I wonder if C++ really would copy in this situation, or if it could swap).

Please document all three functions independently, so they all show up as documented in the Doxygen. If you want to void repetition then I suppose you could fully document one version and have a one-liner for the others that points to the full docs of the one, but personally I'd just add the repetition.